pkg/cli/newapp/newapp.go

```diff
@@ -134,9 +134,15 @@ To list all local templates and image streams, use:
 To search templates, image streams, and Docker images that match the arguments provided, use:
 
   oc new-app -S php
-  oc new-app -S --template=ruby
+  oc new-app -S --template=rails
   oc new-app -S --image-stream=mysql
-  oc new-app -S --docker-image=python
+  oc new-app -S --docker-image=registry.access.redhat.com/ubi8/python-38
+
+For details on how to use the results from those searches, use:
+
+  oc help new-app
+
+To see how to provide images, image streams, templates, or source code locations as inputs into 'oc new-app'.
```

Comment:
final grammar nit  - final line here isn't a complete sentence. Perhaps we can get rid of it?

Reply:
yeah it was suppose to read as a follow on clause to the prior sentence, but splitting up like that does seem cumbersome in hindsight.

I'm actually going to push the change where I combine a portion of this with the prior sentence.  See what you think of that.